tests/model_registry/model_catalog/huggingface/test_huggingface_model_validation.py

```
@@ -34,10 +77,62 @@
 @pytest.mark.usefixtures("updated_catalog_config_map")
 class TestHuggingFaceModelValidation:
     """Test HuggingFace model values by comparing values between HF API calls and Model Catalog api call"""
 
+    def test_huggingface_model_metadata_last_synced(
+        self: Self,
+        current_epoch_time: float,
+        updated_catalog_config_map: tuple[ConfigMap, str, str],
+        model_catalog_rest_url: list[str],
+        model_registry_rest_headers: dict[str, str],
+        expected_catalog_values: dict[str, str],
+        huggingface_api: bool,
```

Ruff (0.14.13)

Unused method argument: `huggingface_api`

(ARG002)